meson.build

```diff
@@ -16,6 +16,7 @@ project(
     'c_args=-Wno-unused-function -Wno-conversion -Wno-misleading-indentation -Wno-incompatible-pointer-types',
     'fortran_args=-Wno-conversion',
     'fortran_std=legacy',
+    'libname=openblas'
```

Comment:
It is in principle possible (if uncommon) for BLAS and LAPACK to not come from the same library. E.g., `blis` provides only BLAS, and then has to be used with another LAPACK library. So I think it'd be better to have two options:
```meson
'blas=openblas',
'lapack=openblas'
```